### ISSUE TITLE
Add an embed widget/wizard

### ISSUE THREAD

**Issue by reefdog** [documentcloud/wordpress-documentcloud#25]
Original version of the plugin had a button (in visual mode) that spawned a wizardy form to let people compose an embed (choosing sidebar yes/no, set size, etc.). It was TinyMCE-based but broke at some point. It's been removed entirely.

At some point we should add something like that back in. Here's what @eyeseast did for another plugin: https://github.com/sunlightlabs/navis-openstates/blob/master/js/tinymce/legislators-tinymce.js

**Comment by reefdog** [comment]
Considering using [Shortcake](https://github.com/wp-shortcake/shortcake) rather than rolling our own. We'd have to say "go install Shortcake if you want a wizard", but I'm okay with that.

**Comment by knowtheory** [comment]
@reefdog so it'd be nice to take a quick poke through and think if it's architecturally sane (or get some advice from others), but yeah, i'd say put that wood behind the same arrow.

**Comment by aschweigert** [comment]
fwiw, there's some talk of shortcake becoming part of wp core but it's hard to say if/when that would happen. i'm catching up with daniel this week and can ask him if you want. otherwise i'd be a little leery re: making it a "requirement", we've looked at it but are kind of taking a wait and see approach for the moment.

**Comment by knowtheory** [comment]
@aschweigert: just as a point of note, there was a security vulnerability in the TinyMCE integration that the plugin had been using, and our solution was to just remove the wizard for the time being (it wasn't working anyway).

Would be interested if Shortcake would be worth rolling out even as a tentative feature since we currently offer no alternative.

**Comment by reefdog** [comment]
Yeah, wouldn't make it a requirement for base functionality, just as a potential answer for wizardy embed option composition.
